vme/src/structs.h

```diff
@@ -227,16 +227,97 @@ class char_data : public unit_data
 };
 
 /* ------------------  PC SPECIFIC STRUCTURES ------------------------ */
-
-struct pc_account_data
+class pc_account_data
 {
-    float credit;        /* How many coin units are left on account?       */
-    ubit32 credit_limit; /* In coin units (i.e. cents / oerer)             */
-    ubit32 total_credit; /* Accumulated credit to date (coin units)        */
-    sbit16 last4;        /* The last four digits of his credit card, or -1 */
-    ubit8 cracks;        /* Crack-attempts on CC last4                     */
-    ubit8 discount;      /* 0 - 100% discount                              */
-    ubit32 flatrate;     /* The expiration date of a flat rate service     */
+public:
+    pc_account_data() = default;
+
+    void readFrom(CByteBuffer &buf, ubit8 unit_version, int &error)
```

Comment:
I hope the functions will move into a .cpp file soon...

Reply:
Most certainly, its easier to have them in header only when rebasing. Each extract PR does that